09-features.md

```diff
@@ -12,5 +12,5 @@ This flags may only be used in the `init` message.
 | Bits | Description                                     | Link                                                                |
 |------|-------------------------------------------------|---------------------------------------------------------------------|
 | 1    | The sending node wishes to announce the channel | [BOLT #7](07-routing-gossip.md#the-announcement_signatures-message) |
-
+| 2/3  | `initial_routing_sync` | The sending node needs a complete routing information dump | [BOLT #7](07-routing-gossip.md#initial-sync) |)
```

Comment:
The format of this new entry doesn't match up with that of the one above. I thin they should be made to be consistent.